proto/translate/v1/translate.proto

```diff
@@ -67,21 +67,21 @@ message DownloadTranslationFileResponse {
 
 message CreateMessagesRequest {
   string service_id = 1;
-  Messages messages = 2;
+  Translation messages = 2;
 }
```

Comment:
```suggestion
message CreateTranslationRequest {
  string service_id = 1;
  Translation translation = 2;
}
```

In other places you should change too, as we do not have messages resource anymore

Reply:
Oki, will change ;)